### ISSUE TITLE
Zoo Filtering options does not carry over when clicking Activity in menu

### ISSUE THREAD

**Issue by PatriciaMoller** [SmartZoos/Toolset#186]
When navigating back to the list of Activities via the menu on top of the page or after editing an Act., whatever filter you had does not carry over. Making the only way to get the filter back is via the search options or going via the Start Page. 

The setting should be retained for when you are taken back to the Act. list again. Otherwise, the user could be faced with a lot of unfamiliar entries.

**Comment by pjotrsavitski** [comment]
I do not suppose such logic is applied in general. Navigating through the menu does in general start from scratch. Example: even if you started on the third page, it would still take to to the listing.

We could have some way to store specific page URL the user navigated from, before opening an editing page and then show some king of element that would allow going back to the same spot.

It should be a specific button with specific functionality. Other solutions would probably create a lot of confusion as the search options would be remembered until the form is reset or the session is closed.

**Comment by pjotrsavitski** [comment]
Yes. We should also define it that should only affect the Zoo filter or the whole search.

**Comment by PatriciaMoller** [comment]
This is mainly an issue when editing Activities and Items. When saving, you get to the Item's info page where the only way back to the list is through the top-menu.

So maybe what is needed is a "Back to list". Either from the Activity's/Item's page, or that by pressing Save you are taken back to the list, with the same filtering options. Then, using the top-menu will remain a "fresh start". Is there any reason why saving leads to the "landing page"?

The Zoo filter part of this issue might be irrelevant if each Zoo will run SmartZoos on their own, as there won't be any other locations' items to filter. However other filters, such as Item Type, Language and Difficulty will still be affected.
